meson.build

```diff
@@ -0,0 +1,17 @@
+project('graph-prototype', 'cpp',
+  version : '0.1',
+  default_options : ['warning_level=3', 'cpp_std=gnu++20'])
+
+fmt_dep = dependency('fmt', version:'8.1.1')
+ut_dep = dependency('boost.ut')
+
+cmake = import('cmake')
+libreflcpp = cmake.subproject('refl-cpp')
+reflcpp_dep = libreflcpp.dependency('refl-cpp')
+
```

Comment:
I think we are missing the pmtv dependency here: https://github.com/fair-acc/graph-prototype/actions/runs/4902739549/jobs/8772423723?pr=49#step:11:12 Probably it gets picked up from the system path on your system, but isn't available on the CI runner. Also it might not have been a dependency when this PR was started :)

Reply:
Added it in - but to get this building with wasm cleanly requires https://github.com/gnuradio/pmt/pull/90 to have the python enable flag work

Reply:
The wrap file had gotten messed up, but is fixed now - @wirew0rm if you don't mind kicking the CI run again

Reply:
Kicked the CI. @mormj will add you to the organisation/repo if you do not mimd. FAIR enough?